src/ec/packages/ec-component-site-header-harmonised/site-header-harmonised.test.js

```diff
@@ -40,7 +41,7 @@ describe('EC - Site Header Harmonised', () => {
   });
 
   describe('Group 2', () => {
-    dataGroup1.group = 'group2';
+    dataGroup2.group = 'group2';
```

Comment:
Ok, this should come from the specs, because if you had to do it here it means that is not there. I see you did it for the group3 the correct way, you should also align the other two groups (well, group1 is the default...so), group2 for instance is set in the story and if you check the rendered markup `yarn check:component site-header-harmonised` you will notice that the class is wrong for group2, cause it gets the default.